code/modules/client/preferences.dm

```diff
@@ -82,6 +82,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 	/// If set to TRUE, will update character_profiles on the next ui_data tick.
 	var/tainted_character_profiles = FALSE
 
+	///Whether the user wants to see body size being shown in the preview
+	var/show_body_size = FALSE
+
 /datum/preferences/Destroy(force, ...)
```

Comment:
Since fixing the character scaling in the UI doesn't seem to be an option (discussed on discord), the best way to handle the issue would be to either implement this as an preferences option, or add some sort of notice saying that the scaling doesn't look as bad in-game.
Both of those would require a relatively sizable refactor though, and none of the other servers with this issue have fixed it either.

So in the interest of actually getting this merged at some point, I'd suggest just leaving it how it is for now and removing this variable until a separate PR can be made.
```suggestion
	var/tainted_character_profiles = FALSE

/datum/preferences/Destroy(force, ...)
```